package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/HierarchyBuilder.cs

```diff
@@ -151,6 +151,7 @@ public struct Query<T> : IQuery where T : SampleBase, new() {
       }
       if (options.importTransforms) {
         FindPathsJob.queries[7] = (FindPathsJob.IQuery)new FindPathsJob.Query<XformSample>();
+        FindPathsJob.queries[8] = (FindPathsJob.IQuery)new FindPathsJob.Query<ScopeSample>();
```

Comment:
We may need to import scopes regardless of transform import state... I don't know if there is a use case, but I could imagine making a hierarchy of scopes, disabling `importTransforms` and then being confused why my scopes are showing up.